source/common/config/BUILD

```diff
@@ -8,6 +8,17 @@ load(
 
 envoy_package()
 
+envoy_cc_library(
+    name = "api_type_db_lib",
```

Comment:
Note that I'm not committed to API type DB, @lizan and I have discussed alternatives, but we're going to need some module for mapping types across versions, so let's land this for now and we can rename later.

Reply:
Sorry what do you mean by this exactly? You mean whether we will embed the type DB directly versus reflection mechanisms?

Reply:
Yep, what you said.